sw/device/lib/base/memory.c

```diff
@@ -36,13 +36,13 @@ void *memset(void *dest, int value, size_t len) {
 }
 #endif  // !defined(HOST_BUILD)
 
-#if !defined(HOST_BUILD)
 enum {
   kMemCmpEq = 0,
   kMemCmpLt = -42,
   kMemCmpGt = 42,
 };
 
+#if !defined(HOST_BUILD)
```

Comment:
`ifndef`?

Reply:
Hmm, the rest of the file has `if !defined`  (in fact, this line is just moved, not added) -- should I change the whole file?